### DISCUSSION TITLE
feat: Youtube Support

### DISCUSSION THREAD

**Discussion by hassanrehman01398** [steven-tey/novel#328] [Ideas]
### Describe the feature you'd like to request

I wish Novel would support youtube iframes also.

### Describe the solution you'd like to see

While pasting the link of youtube, the link should be changes into iframe.

**Comment by andrewdoro** [comment]
I suppose this would already work. Did you had any problems with this Tiptap Extension? I am not sure if Markdown would break your link on pasting.
https://tiptap.dev/docs/editor/api/nodes/youtube

**Comment by MODSetter** [comment]
Added a basic support for this here: https://github.com/steven-tey/novel/pull/386
Lets see if it gets approved.
